components/alert/w-alert.vue

```diff
@@ -1,5 +1,41 @@
+<script setup>
+import { computed } from 'vue';
+import { wExpandTransition } from '#generics';
+import { createModel, modelProps } from 'create-v-model';
+import { alert } from '@warp-ds/component-classes';
+
+const props = defineProps({
+  title: String,
+  role: {
+    type: String,
+    default: 'alert'
+  },
+  negative: Boolean,
+  positive: Boolean,
+  warning: Boolean,
+  info: Boolean,
+  ...modelProps(),
+});
+
+const possibleTypeProps = ['negative', 'positive', 'warning', 'info'];
+
+const emit = defineEmits(['update:modelValue']);
+const model = createModel({ props, emit });
+const alertColorType = computed(() => possibleTypeProps.find(e => props[e]));
+const activeWrapperClassNames = computed(() => alert[alertColorType.value]);
+const activeIconClassNames = computed(() => alert[`${alertColorType.value}Icon`]);
+const wrapperClass = computed(() => [
+  alert.alert,
+  activeWrapperClassNames.value
+]);
+const iconClass = computed(() => [
+  alert.icon,
+  activeIconClassNames.value
+]);
+</script>
+
 <template>
-  <div class="f-expandable">
+  <div class="will-change-height">
```

Comment:
I see that "will-change-height" classname is assigned to component-classes' [`expandable` object](https://github.com/warp-ds/component-classes/blob/alpha/index.js#L194). Not sure it makes sense to apply it here, too, but thought I'd mention it.

Reply:
Not sure if we really want to use it at all after having read this (and other things) in MDN:

> Don't apply will-change to elements to perform premature optimization. If your page is performing well, don't add the will-change property to elements just to wring out a little more speed. will-change is intended to be used as something of a last resort, in order to try to deal with existing performance problems.
😄

But I think we will just keep it for now?

Reply:
Sounds like something we could add to the "will improve later" task pile 😄 I don't think we have a pile like that ready in Jira, so maybe add a TODO comment for now? We can eventually scan our repos to find all TODOs and then create tasks based on those.

Still, the question remains: what about using `ccExpandable.expandable` instead of a hard-coded utility class? What do you think about that?

Reply:
Ah ok so NOW I understand the question. 😆 😅 
For some reason I think this only exists in Vue... So maybe it would be ok just to use the class from ccExpandable here. 👍

Reply:
Ah no... at second thought, that wouldn't be right. 😄 
If you decide to add more classes to or remove `will-change-height` from ccExpandable that too would affect this component and that might not be the desired result.

As this is only in Vue I think it would be better just to leave it as it is, with a todo comment, and not pollute the component-classes with this by adding it just because of the use here in Vue, that might not be correct? 😄 Agree? 🙂

Reply:
Sounds good!